snap/snapcraft.yaml

```diff
@@ -71,21 +72,41 @@ parts:
       - libqrencode-dev
     stage-packages:
       - ca-certificates
-      - libboost-chrono1.58.0
-      - libboost-test1.58.0
-      - libboost-thread1.58.0
-      - libevent-2.0-5
-      - libevent-core-2.0-5
-      - libevent-pthreads-2.0-5
+      - libboost-chrono1.65-dev
+      - libboost-test1.65-dev
+      - libboost-thread1.65-dev
+      - libboost-filesystem1.65.1
+      - libevent-2.1-6
+      - libevent-core-2.1-6
+      - libevent-pthreads-2.1-6
       - libminiupnpc10
-      - libprotobuf9v5
+      - libprotobuf-c-dev
+      - libprotobuf10
       - libqrencode3
       - libqt5test5
-      - libsodium18
+      - libsodium-dev
       - libzmq5
     after:
       - berkeleydb
-      - desktop-qt5
+      - qt
+  qt:
+    source: https://github.com/ubuntu/snapcraft-desktop-helpers.git
+    source-subdir: qt
```

Comment:
I think it'd be good to add a `source-commit` line so build doesn't change when the upstream branch is updated.

Reply:
How would we track if there were security fixes?

Reply:
I think it's just a tradeoff. Disadvantage of pinning is we have to manually bump the version to get security updates. Advantages of pinning are that it gives us a way to audit changes to the upstream repository, and it can prevent bugs or incompatibilities introduced in the upstream repository from automatically showing up in our builds without us changing anything.

Reply:
Our deterministic builds are also built from pinned versions, so at least it would mimic our existing release process.